ext/js/language/text-scanner.js

```diff
@@ -770,6 +774,7 @@ export class TextScanner extends EventDispatcher {
 
         const inputInfo = this._getMatchingInputGroupFromEvent('touch', 'touchEnd', e);
         if (inputInfo === null || inputInfo.input === null) { return; }
+        if (touchReleaseTime - this._touchPressTime < inputInfo.input.minimumTouchTime) { return; }
```

Comment:
I think you need to add parens to the subtract https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence

Reply:
According to the table shown there (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table) the subtraction operator is higher precedence than the less than operator. The intended result here is to subtract first.